src/realm/sync/tools/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.6)
+
+add_executable(InspectorInspectClientRealm "inspect_client_realm.cpp")
+set_target_properties(InspectorInspectClientRealm PROPERTIES OUTPUT_NAME "realm-inspect-client-realm")
```

Comment:
Why don't the top two tools do the DEBUG_POSTFIX thing? I would expect either all or neither to do it.